gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/reset_password.html

```diff
@@ -60,6 +60,10 @@
                 <p th:if="${passwordSettings.excludePasswordsInDictionary}" id="excludePasswordsInDictionary" class="advice">Don't use common names or passwords</p>
                 <p th:if="${passwordSettings.excludeUserProfileInfoInPassword}" id="excludeUserProfileInfoInPassword" class="invalid">Don't use your profile information in
                     password</p>
+                <p id="matchPasswords" class="invalid">Passwords match</p>
```

Comment:
"Password match" text should be in the `message_en.properties` and `message_fr.properties`.  There are 2 properties files for both en and fr. You need to add the property in all 4 files. Rebase this branch to master; resolve the conflicts and you should be able to all the existing properties for the `reset_password` template.